nilearn/maskers/base_masker.py

```diff
@@ -396,3 +396,33 @@ def _check_fitted(self):
                 "has not been fitted. "
                 "You must call fit() before calling transform()."
             )
+
+
+class _BaseSurfaceMasker(TransformerMixin, CacheMixin, BaseEstimator):
+    """Class from which all surface maskers should inherit."""
```

Comment:
Maybe a "version added" tag? But I guess it doesn't really matter because it wont be public anyway

Reply:
+addition in Whatsnew

Reply:
hmmm not sure given this is a private class
I'd wait before adverstising it